platform-api/src/routers/discord.py

```
@@ -0,0 +1,77 @@
+        "timestamp": datetime.now(timezone.utc).isoformat(),
+    }
+    if body.fields:
+        embed["fields"] = [f.model_dump() for f in body.fields]
+
+    # 4. Send (Sync or Async)
+    if body.async_mode:
+        # Fire-and-forget: Return 202 immediately, send in background
+        # TODO: Replace asyncio.create_task with ctx.waitUntil for Cloudflare Workers
+        # In Workers, asyncio tasks will be terminated when response returns.
+        # Proper fix: wire request.scope["ctx"] and use ctx.waitUntil(post_webhook(...))
+        asyncio.create_task(post_webhook(webhook_url, embed))
```

Ruff (0.15.11)

[warning] Store a reference to the return value of `asyncio.create_task`

(RUF006)